.github/workflows/build_test_deploy.yml

```diff
@@ -1602,7 +1602,7 @@ jobs:
 
       - name: Build
         if: ${{needs.build.outputs.docsChange == 'nope'}}
-        run: turbo run build-wasm --cache-dir=".turbo" -- --target nodejs --dev
+        run: turbo run build-wasm --cache-dir=".turbo" -- --features tracing/release_max_level_info --target nodejs --dev
```

Comment:
Should the dev build have tracing enabled?

Reply:
I don't think so